docs/getting-started/raster/services/export_api.md

```diff
@@ -1,40 +1,46 @@
+- file extension: ".gpkg"
+
+#### Metadata
+JSON file that contains the record metadata such as footprint and resolution
+- file extension: ".JSON"
+- provided fields follow the profile convention [here](/catalog-information/v1_0/raster_profile)
 
-1. Request Parameters - The POST message can receive the following request parameters:
-    1. dbId - Raster Catalog Record DB ID
-    2. bbox - one of the following types (BBOX / GeoJson Geometry (Polygon / MultiPolygon), empty)
-        3. Bounding box corners (lower left, upper right)=[minx,miny,maxx,maxy] in crs units as array. 
-        4. GeoJSon geometry of Polygon or MultiPolygon type
-        5. If empty - original layer's footprint is taken.
-    3. targetResolution - The target resolution in which the tiles will be created - (max DEGREE to PIXEL)
-    4. callbackURLs - webhook to call when GPKG creation is complete
-    5. crs - Target CRS for the GPKG to be created with (currently only "EPSG:4326" is supported)
-    6. priority - Optional field to set the priority of the export task
-2. Response Parameters - The POST message can return the following response parameters:
-    1. json object for newly created or "In progress" export task
-        - status: "In-Progress" - Status of the export task
-        - id: uuid - unique identifier for export request
-        - taskIds: uuid[] -  unique identifier for inner tasks of export request
-    2. json object for completed task
-        - status: "Completed" - Status of the export task
-        - requestId: uuid - unique identifier for export request - the field that was returned from orginal create request
-        - fileUri: string - download link for the exported GPKG file
-        - expirationTime: string($date) - date when the exported file will be deleted
-        - fileSize: number - GPKG file size in bytes
-        - dbId: uuid - Raster Catalog Record DB ID - from orginal request
-        - packageName: string - the GPKG file name
-        - bbox: one of the following types (BBOX / GeoJson Geometry (Polygon / MultiPolygon), empty) - from orginal request
-        - targetResolution - The target resolution in which the tiles will be created - (max DEGREE to PIXEL) - from orginal request
-        - success: boolean - whether the export task was successfull
-        - errorReason: string - if export task was unsuccessful this field will describe the error
-3. Callback response object:
-    - json object for newly created or "In progress" export task
+
+
+### Important Notes
+1. The data is returned via callback response or on request response (structure in appendix 1)
+2. To download the data one needs to use authentication token
+3. To download the "metadta.json" file, write the GPKG url and change the sufix to ".json"
+
+
+### Appendix 1 - Callback response structure
     - requestId: uuid - unique identifier for export request - the field that was returned from orginal create request
```

Comment:
for the callback, I guess you send a JSON body, I would expect that you place your JSON schema here

Reply:
added